packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-checkbox-group/example.html

```diff
@@ -0,0 +1,238 @@
+      <span slot="error">Subject to availibility</span>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="food-1" value="spaghetti" />
+      <label slot="label">Spaghetti</label>
+      <span slot="help">This is a help message</span>
+    </gux-form-field-checkbox>
+  </gux-form-field-checkbox-group-beta>
+</form>
+
+<h2>Example with Group Error</h2>
+<form onchange="notify(event)" oninput="notify(event)">
+  <gux-form-field-checkbox-group-beta>
+    <label slot="group-label">Food</label>
+
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="food-1" value="pizza" />
+      <label slot="label">Pizza</label>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input
+        slot="input"
+        type="checkbox"
+        name="food-1"
+        value="sandwich"
+        disabled
+      />
+      <label slot="label">Sandwich</label>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="food-1" value="sushi" />
+      <label slot="label">Sushi</label>
+      <span slot="error">Subject to availibility</span>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="food-1" value="spaghetti" />
+      <label slot="label">Spaghetti</label>
+      <span slot="help">This is a help message</span>
+    </gux-form-field-checkbox>
+    <span slot="group-error">Subject to availibility</span>
+  </gux-form-field-checkbox-group-beta>
+</form>
+
+<h2>Example with Group Help text</h2>
+<form onchange="notify(event)" oninput="notify(event)">
+  <gux-form-field-checkbox-group-beta>
+    <label slot="group-label">Food</label>
+
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="food-1" value="pizza" />
+      <label slot="label">Pizza</label>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input
+        slot="input"
+        type="checkbox"
+        name="food-1"
+        value="sandwich"
+        disabled
+      />
+      <label slot="label">Sandwich</label>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="food-1" value="sushi" />
+      <label slot="label">Sushi</label>
+      <span slot="error">Subject to availibility</span>
+    </gux-form-field-checkbox>
+
+    <gux-form-field-checkbox>
+      <input slot="input" type="checkbox" name="food-1" value="spaghetti" />
+      <label slot="label">Spaghetti</label>
+      <span slot="help">This is a help message</span>
+    </gux-form-field-checkbox>
+    <span slot="group-help">This is a group help message</span>
+  </gux-form-field-checkbox-group-beta>
+</form>
+
+<h2>Example with disabled group</h2>
+<form onchange="notify(event)" oninput="notify(event)">
+  <gux-form-field-checkbox-group-beta disabled>
+    <span slot="group-label">Food</span>
```

Comment:
Sorry I did not spot this yesterday but one last thing, I think this needs to be a `label` tag as currently the `group-label` does not display.